tests/integration/models/release.spec.coffee

```diff
@@ -115,3 +122,36 @@ describe 'Release Model', ->
 							build_log: 'web log'
 						]
 
+		describe.only 'resin.models.release.tags', ->
```

Comment:
Don't commit this `.only`! This disables all of the rest of the test suite

Reply:
Oh my...